phpunit.xml.dist

```diff
@@ -24,4 +24,7 @@
             </exclude>
         </whitelist>
     </filter>
+    <php>
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="weak" />
```

Comment:
and if you insist on testing on unmaintained Symfony version, Travis should set the env variable only for these jobs in the matrix, not for all jobs (so that tests break if the bundle is using a deprecated API)